third_party/externs/ts/gapi/bigquery.d.ts

```diff
@@ -60,13 +61,48 @@ declare namespace gapi.client {
       projectId: string;
     }
 
+    interface GetTableRequest {
+      datasetId: string;
+      projectId: string;
+      tableId: string;
+    }
+
     interface ListTablesResponse {
       kind: string;   // Should always be "bigquery#tableList"
       etag: string;
       nextPageToken: string;
       tables: Array<TableResource>;
     }
 
+    interface BigqueryTable {
```

Comment:
Renamed to just "Table".
This object is not the response though, it represents an actual table. A response would be a thin wrapper around it with its `body` property set to this object.